lib/services/mail_service.dart

```diff
@@ -0,0 +1,98 @@
+import 'dart:convert';
+import 'dart:io';
+import 'dart:typed_data';
+
+import 'package:enough_mail/enough_mail.dart';
+import 'package:summer2022/exceptions/fetch_mail_exception.dart';
+import 'package:summer2022/models/Digest.dart';
+
+/// todo: remove this once mailpiece class is properly implemented
+class MailPiece {
+  String ID;
+  String EmailID;
+  DateTime TimeStamp = new DateTime(2022, 1, 1);
+  String Sender;
+  String MidID;
+  String ImageText;
```

Comment:
Canonical Dart uses `lowerCamelCase` for variable names.
See: https://dart.dev/guides/language/effective-dart/style#do-name-other-identifiers-using-lowercamelcase